packages/malloy/src/dialect/postgres.ts

```diff
@@ -61,8 +61,11 @@ export class PostgresDialect extends Dialect {
   divisionIsInteger = true;
   functionInfo: Record<string, FunctionInfo> = {};
 
-  quoteTableName(tableName: string): string {
-    return `${tableName}`;
+  quoteTablePath(tablePath: string): string {
+    return tablePath
+      .split(".")
+      .map((part) => `"${part}"`)
```

Comment:
I debated doing "add quotes if it doesn't currently have them", like:

```ts
      .map((part) => {
        if (part.startsWith('"')) {
          return part;
        }
        return `"${part}"`;
      })
 ```
 
 but unclear if it's needed